released/packages/coq-mathcomp-ssreflect/coq-mathcomp-ssreflect.1.10.0/opam

```diff
@@ -8,7 +8,7 @@ license: "CeCILL-B"
 
 build: [ make "-C" "mathcomp/ssreflect" "-j" "%{jobs}%" ]
 install: [ make "-C" "mathcomp/ssreflect" "install" ]
-depends: [ "coq" { ((>= "8.7" & < "8.12~") | (= "dev"))} ]
```

Comment:
It seems you removed one parenthesis too much!

Reply:
Good catch! Fixed.